vault/resource_ssh_secret_backend_role_test.go

```diff
@@ -137,6 +139,26 @@ func TestAccSSHSecretBackendRoleOTP_basic(t *testing.T) {
 	})
 }
 
+func TestAccSSHSecretBackendRole_template(t *testing.T) {
+	backend := acctest.RandomWithPrefix("tf-test/ssh")
+	name := acctest.RandomWithPrefix("tf-test-role")
+	resource.Test(t, resource.TestCase{
+		Providers:    testProviders,
+		PreCheck:     func() { testutil.TestAccPreCheck(t) },
+		CheckDestroy: testAccSSHSecretBackendRoleCheckDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSSHSecretBackendRoleConfig_template(name, backend),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("vault_ssh_secret_backend_role.test_role", "name", name),
+					resource.TestCheckResourceAttr("vault_ssh_secret_backend_role.test_role", "backend", backend),
+					resource.TestCheckResourceAttr("vault_ssh_secret_backend_role.test_role", "default_user", "ssh-{{identity.entity.id}}-user"),
```

Comment:
should we also assert
```
resource.TestCheckResourceAttr("vault_ssh_secret_backend_role.test_role", "default_user_template", "true)
```

Reply:
`default_user_template` is not returned by the vault API

Reply:
Oops I was wrong. `default_user_template` is returned by the API